app/src/main/java/com/hms/referenceapp/photoapp/ui/listuser/ListUserFragment.kt

```diff
@@ -87,6 +75,18 @@ class ListUserFragment : DialogFragment() {
         listUserAdapter.onUserSelectListener {
             viewModel.selectUser(it)
         }
+        binding.saveValuesButton.setOnClickListener {
```

Comment:
you can override setupListeners function and move listeners inside it if it's possible